content/en/synthetics/api_tests/_index.md

```diff
@@ -93,6 +93,23 @@ Define the request you want to be executed by Datadog:
 [3]: /synthetics/private_locations/
 {{% /tab %}}
 
+{{% tab "DNS Test" %}}
+
+{{< img src="synthetics/api_tests/dns_test.png" alt="Define DNS Request"  style="width:80%;" >}}
+
+1. **Choose request type**: `DNS`
```

Comment:
```suggestion
1. **Choose a request type**: `DNS`.
```

Reply:
I guess we should also change for HTTP, SSL, and TCP tests then?

Reply:
I changed for all types.